protocol/xmrmaker/offers/offer_manager.go

```diff
@@ -100,10 +98,12 @@ func (m *Manager) AddOffer(
 		return nil, err
 	}
 
+	// TODO: Is it a problem that we don't save the relayerFee to the database?
+	//       Should we place the relayer fee directly in the offer?
```

Comment:
```suggestion
	// TODO: Is it a problem that we don't save `useRelayer` to the database?
	//       Should we place `useRelayer` directly in the offer?
```

I'm wondering if it makes more sense to have swapd take a `--use-relayer` flag rather than each individual offer. what do you think? either way doesn't need to be addressed in this PR

Reply:
I'll remove this TODO. With the latest version of the PR, the only reason to use the `--use-relayer` flag is for testing. It will force the use of the relayer, even when Bob has sufficient ETH to pay for the claim. But that use case doesn't make financial sense, nor does it provide additional privacy. I think it is fine if `swapd` forgets that the flag was given when the original offer was made on restart. I'll also tag the flag as hidden so end users are not tempted to use it and pay additional fees by mistake.

 Using it doesn't make financial sense or provide additional privacy if you have enough ETH in the account being claimed into, and it is used automatically if Bob doesn't have enough ETH.